scss/_pagination.scss

```
@@ -1,10 +1,12 @@
 .pagination {
   // scss-docs-start pagination-css-vars
   --#{$prefix}pagination-padding-x: #{$pagination-padding-x};
+  --#{$prefix}pagination-label-padding-inline-x: #{$pagination-label-padding-inline-x};
```

```suggestion
  --#{$prefix}pagination-label-padding-x: #{$pagination-label-padding-x}; // Boosted mod
```

Just wondering if we should keep the `inline` or not.

---

I would say no, especially since we removed the `padding-inline` property.